specification/computeschedule/resource-manager/Microsoft.ComputeSchedule/preview/2024-06-01-preview/computeschedule.json

```diff
@@ -0,0 +1,1348 @@
+          {
+            "name": "Availability",
+            "value": "Availability",
+            "description": "Optimize while considering availability of resources"
+          },
+          {
+            "name": "CostAvailabilityBalanced",
+            "value": "CostAvailabilityBalanced",
+            "description": "Optimize while considering a balance of cost and availability"
+          }
+        ]
+      }
+    },
+    "ResourceOperation": {
+      "type": "object",
+      "description": "High level response from an operation on a resource",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "Unique identifier for the resource involved in the operation, eg ArmId"
+        },
+        "errorCode": {
+          "type": "string",
+          "description": "Resource level error code if it exists"
+        },
+        "errorDetails": {
+          "type": "string",
+          "description": "Resource level error details if they exist"
+        },
+        "operation": {
+          "$ref": "#/definitions/ResourceOperationDetails",
+          "description": "Details of the operation performed on a resource"
+        }
+      },
+      "required": [
+        "resourceId"
+      ]
+    },
+    "ResourceOperationDetails": {
+      "type": "object",
+      "description": "The details of a response from an operation on a resource",
+      "properties": {
+        "operationId": {
+          "type": "string",
+          "description": "Operation identifier for the unique operation"
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Unique identifier for the resource involved in the operation, eg ArmId"
+        },
+        "opType": {
+          "type": "string",
+          "description": "Type of operation performed on the resources",
+          "default": "Unknown",
+          "enum": [
+            "Unknown",
+            "Start",
+            "Deallocate",
+            "Hibernate"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceOperationType",
+            "modelAsString": true,
+            "values": [
+              {
+                "name": "Unknown",
+                "value": "Unknown",
+                "description": "The default value for this enum type"
+              },
+              {
+                "name": "Start",
+                "value": "Start",
+                "description": "Start operations on the resources"
+              },
+              {
+                "name": "Deallocate",
+                "value": "Deallocate",
+                "description": "Deallocate operations on the resources"
+              },
+              {
+                "name": "Hibernate",
+                "value": "Hibernate",
+                "description": "Hibernate operations on the resources"
+              }
+            ]
+          },
+          "x-ms-enum.modelAsString": true
+        },
+        "subscriptionId": {
```

Comment:
>subscriptionId [](http://example.com/codeflow?start=9&length=14)

"format": "uuid" ?

Reply:
Using the uuid made the validation for uuid fail, for now, I'd like to keep these as strings.